github/workflows/node.js.yaml

```diff
@@ -0,0 +1,70 @@
+name: Node.js CI/CD
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
```

Comment:
shouldnt we only deploy on push, and not pull request?

Reply:
I think we need both, PR commits don't count as pushes I think. That would be directly pushing code from the branch (through CLI), which I have disabled for the repo.